code/__HELPERS/maths.dm

```diff
@@ -85,3 +85,18 @@ var/gaussian_next
 		gaussian_next = R2 * working
 	return (mean + stddev * R1)
 #undef ACCURACY
+
+
+
+// oof, what a mouthful
+// Used in status_procs' "adjust" to let them modify a status effect by a given
+// amount, without inadverdently increasing it in the wrong direction
+/proc/directional_bounded_sum(orig_val, modifier, bound_lower, bound_upper)

```

Comment:
It's so I can define the operation used in `AdjustSTATUS` with bounds that don't cause it to change in the wrong direction, without having to write the same code like 20 times